src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder results = new StringBuilder();
+        String importantSymbol = ",";
+        int operationTypeIndex = 0;
+        int amountIndex = 1;
+        int arraysSize = 3;
+        int resultIndex = 2;
+        String[] type = new String[arraysSize];
+        int[] amount = new int[arraysSize];
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String value = reader.readLine();
+            while (value != null) {
+                String[] values = value.split(importantSymbol);
+                for (int i = 0; i < arraysSize; i++) {
```

Comment:
The loop over `arraysSize` assumes there will always be three types of operations. This is not flexible and might not work if the file contains more types of operations. Consider using a map or another data structure that can dynamically handle an arbitrary number of operation types.